x-pack/plugin/ccr/src/main/java/org/elasticsearch/xpack/ccr/action/TransportResumeFollowAction.java

```diff
@@ -320,21 +325,37 @@ private static ShardFollowTask createShardFollowTask(
         return historyUUIDs.split(",");
     }
 
-    private static final Set<Setting<?>> WHITE_LISTED_SETTINGS;
+    /**
+     * These are settings that are not replicated to the follower index and
+     * therefor these settings are not validated whether they have the same
+     * value between leader and follower index.
+     *
+     * These dynamic settings don't affect how documents are indexed (affect index time text analysis) and / or
+     * are inconvenient if they were replicated (e.g. changing number of replicas).
+     */
+    static final Set<Setting<?>> WHITE_LISTED_SETTINGS;
```

Comment:
I think this should be renamed to `NON_REPLICATED_SETTINGS`.

Reply:
Agree for a follow-up.